frontend/src/reviews/ReviewCommentDisplay.tsx

```diff
@@ -25,6 +42,22 @@ export default function ReviewCommentDisplay({ response, onReplyButtonClick }: R
     }
   }
 
+  const handleEditOnClick = () => {
+    setUpdatedComment(response.comment || '')
+    setIsEditMode(true)
+  }
+
+  const handleEditOnSave = async () => {
+    setEditCommentErrorMessage('')
+    const res = await patchResponse(response['_id'], updatedComment)
```

Comment:
Minor but can we use the following syntax here for the ID? `response._id`

I'm guessing you've done it this way to circumvent our typing being off here, so I think we'll also need to add `_id` to the `ResponseInterface`